python-backend/app/api/services/NRIS_API_service.py

```diff
@@ -0,0 +1,160 @@
+import decimal
+import uuid
+import requests
+import json
+import functools
+
+from dateutil.relativedelta import relativedelta
+from datetime import datetime, timedelta
+from flask import request, current_app
+from flask_restplus import Resource, reqparse
+from werkzeug.datastructures import FileStorage
```

Comment:
The reliance on the werkzeug datastructures might be an issue. I can't load uwsgi and werkzeug onto my Windows dev environment. It would be nice to limit dependencies on the hosting platform.

Reply:
Are the werkzeug dependencies even being used in this service?

Reply:
removed, and other unused imports.